contracts/Comptroller.sol

```diff
@@ -919,6 +955,81 @@ contract Comptroller is
         }
     }
 
+    /**
+     * @notice Executes a flashLoan operation with the specified assets and amounts.
+     * @dev Transfer the specified assets to the receiver contract and ensures that the total repayment (amount + fee)
+     *      is returned by the receiver contract after the operation for each asset. The function performs checks to ensure the validity
+     *      of parameters, that flashLoans are enabled for the given assets, and that the total repayment is sufficient.
+     *      Reverts on invalid parameters, disabled flashLoans, or insufficient repayment.
+     * @param receiver The address of the contract that will receive the flashLoan and execute the operation.
+     * @param assets The addresses of the assets to be loaned.
+     * @param amounts The amounts of each asset to be loaned.
+     * @param param The bytes passed in the executeOperation call.
+     * @custom:requirements
+     *      - `assets.length` must be equal to `amounts.length`.
+     *      - `assets.length` and `amounts.length` must not be zero.
+     *      - The `receiver` address must not be the zero address.
+     *      - FlashLoans must be enabled for each asset.
+     *      - The `receiver` contract must repay the loan with the appropriate fee.
+     * @custom:reverts
+     *      - Reverts with `InvalidFlashLoanParams()` if parameter checks fail.
+     *      - Reverts with `FlashLoanNotEnabled(asset)` if flashLoans are disabled for any of the requested assets.
+     *      - Reverts with `ExecuteFlashLoanFailed` if the receiver contract fails to execute the operation.
+     *      - Reverts with `InsufficientReypaymentBalance(asset)` if the repayment (amount + fee) is insufficient after the operation.
+     */
+    function executeFlashLoan(
+        address receiver,
+        VTokenInterface[] calldata assets,
+        uint256[] calldata amounts,
```

Comment:
It would be more appropriate to rename `assets` to `vTokens` and `amounts` to `underlyingAmounts`

Reply:
https://github.com/VenusProtocol/isolated-pools/pull/460/commits/79f792d23be54291c106d74909366b0d87f5fd22